easybuild/easyconfigs/l/LuaJIT/LuaJIT-2.0.2-GCC-4.9.2.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'ConfigureMake'
+
+name = "LuaJIT"
+version = "2.0.2"
+
+homepage = "http://luajit.org/"
+description = """LuaJIT is a Just-In-Time Compiler (JIT) for the Lua
+programming language. Lua is a powerful, dynamic and light-weight programming
+language. It may be embedded or used as a general-purpose, stand-alone
+language. """
+
+toolchain = {'name': 'GCC', 'version': '4.9.2'}
+
+sources = [SOURCE_TAR_GZ]
+skipsteps = ['configure']
+source_urls = ['http://luajit.org/download/']
+installopts = 'PREFIX=%(installdir)s'
```

Comment:
@tvandera: we try to maintain more or less the same order of parameters in easyconfigs, which makes them significantly easier to parse (for humans)

Can you please reorder things a bit, as follows?

```python
toolchain = {'name': 'GCC', 'version': '4.9.2'}

source_urls = ['http://luajit.org/download/']
sources = [SOURCE_TAR_GZ]

dependencies = [('ncurses', '5.9')]

skipsteps = ['configure']
installopts = 'PREFIX=%(installdir)s'
```

Looks perfect otherwise, we'll get this tested as soon as you've fixed this minor remark (just push an extra commit to your branch to update this PR).